src/redux/books/ConfigureStore.js

```diff
@@ -0,0 +1,13 @@
+
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import bookReducer from './Books';
+import categoriesReducer from '../Categories/Categories';
```

Comment:
- [ ] Kindly ensure the file imports here match the actual file names:

  ```
  import bookReducer from './books';
  import categoriesReducer from '../categories/categories';
  ```

Reply:
fixed

Reply:
@julie-ify I made all the changes you requested but it didn't work on my local so I had to change them back so I am not really sure what to do